doc/source/testing_pull_requests.rst

```diff
@@ -0,0 +1,226 @@
+Testing an python-for-android pull request
+==========================================
+
+In order to test a pull request, we recommend to consider the following points:
+
+  #. of course, check if the overall thing makes sense
+  #. is the CI passing? if not what specifically fails
+  #. is it working locally at compile time?
+  #. is it working on device at runtime?
```

Comment:
@AndreMiras, I also adapted your approval comment in here, as the introduction of the whole thing :wink:

let me know if it's ok

**Thanks!!** :smile: